packages/plugin-lighthouse/CONTRIBUTING.md

```diff
@@ -15,17 +15,22 @@ To test lighthouse properly we work with a predefined testing setup.
 
 On some OS there could be a problem finding the path to Chrome.
 
-We try to detect it automatically in the set-setup script.
+We try to detect it automatically in the [set-setup script](https://github.com/code-pushup/cli/blob/f11f8a70e1ca4e7fc879fb7b658e8523bb4ee575/testing/test-setup/src/lib/chrome-path.setup.ts).
```

Comment:
Presumably you want to link to the current source code:

```suggestion
We try to detect it automatically in the [`chrome-path.setup.ts` script](../../testing/test-setup/src/lib/chrome-path.setup.ts).
```

Reply:
Yes!! Thx!